frontend/src/scenes/session-recordings/saved-playlists/SavedSessionRecordingPlaylists.tsx

```diff
@@ -154,10 +162,15 @@ export function SavedSessionRecordingPlaylists({ tab }: SavedSessionRecordingPla
             dataIndex: 'pinned',
             render: function Render(pinned, { short_id }) {
                 return (
-                    <LemonButton
+                    <AccessControlledLemonButton
```

Comment:
I think at some point we should consider just making these optional properties on the standard LemonButton

Reply:
https://github.com/PostHog/posthog/pull/37452 working on a PR here for it